cpp/core/shuffle/rss/CelebornPartitionWriter.cc

```diff
@@ -55,24 +55,28 @@ arrow::Status CelebornPartitionWriter::evict(
     Evict::type evictType,
     bool reuseBuffers,
     bool hasComplexType) {
-  rawPartitionLengths_[partitionId] += inMemoryPayload->getBufferSize();
+  int64_t spillTime{0};
+  {
+    ScopedTimer timer(&spillTime);
```

Comment:
@ulysses-you  This should be ```&spillTime```, which has been modified. ```spillTime``` will record the time of each ```evict```, and then accumulate and record the total time through ```spillTime_```.

Reply:
I did not find the difference.. Each evict would call this method, and the previous spillTime_ would record the method elapsed time.

Reply:
The difference is that before, ```spillTime_``` only recorded the time of one evict after the shuffle write was completed, but after the modification, it recorded the total time of evict.